scripts/generator/index.js

```diff
@@ -1,6 +1,37 @@
 const fs = require("fs");
 const handlebars = require('handlebars');
 
+// Handy string extensions
+
+function capitalLetterFunc(){
+    return this.charAt(0).toUpperCase() + this.slice(1)
```

Comment:
Instead of concatenating with `+`, use template literals
```suggestion
    return `${this.charAt(0).toUpperCase()}${this.slice(1)}`
```